docs/manage/software/deploy-software.md

```diff
@@ -10,279 +10,88 @@ languages: []
-    You must implement any required API methods for the chosen component.
-
-{{% /tablestep %}}
-{{% tablestep %}}
-**2. Add your control logic**
-
-When your new model gets added to your machine, its `reconfigure()` method gets called.
-If you want your control logic to run in a loop in the background, you can start this loop here.
-Be sure to also implement logic to handle subsequent calls to the reconfigure method gracefully.
-
-For example, in Python, start your logic in the `reconfigure()` method of <FILE>src/main.py</FILE>:
-
-```python {class="line-numbers linkable-line-numbers" data-line="20-28"}
-# Add these imports
-import asyncio
-from threading import Event
-from viam.logging import getLogger
-
-
-LOGGER = getLogger("control-logic")
-
-
-class ControlLogic(Generic, EasyResource):
-    MODEL: ClassVar[Model] = Model(
-        ModelFamily("naomi", "my-control-logic"), "control-logic"
-    )
-    running = None
-    task = None
-    event = Event()
-
-    # Other methods omitted for brevity
-
-    def reconfigure(
-        self, config: ComponentConfig,
-        dependencies: Mapping[ResourceName, ResourceBase]
-    ):
-        # starts automatically
-        if self.running is None:
-            self.start()
-        else:
-            LOGGER.info("Already running control logic.")
-
-    def start(self):
-        loop = asyncio.get_event_loop()
-        self.task = loop.create_task(self.control_loop())
-        self.event.clear()
-
-    def stop(self):
-        self.event.set()
-        if self.task is not None:
-            self.task.cancel()
-
-    async def control_loop(self):
-        while not self.event.is_set():
-            await self.on_loop()
-            await asyncio.sleep(0)
-
-    async def on_loop(self):
-        try:
-            LOGGER.info("Executing control logic")
-            # TODO: ADD CONTROL LOGIC
-
-        except Exception as err:
-            LOGGER.error(err)
-        await asyncio.sleep(1)
-
-    def __del__(self):
-        self.stop()
-
-    async def close(self):
-        self.stop()
+Then, [configure any hardware and software resources](/operate/get-started/supported-hardware/) that you want to use with your machine and drive with your control logic.
 
-    async def do_command(
-        self,
-        command: Mapping[str, ValueTypes],
-        *,
-        timeout: Optional[float] = None,
-        **kwargs
-    ) -> Mapping[str, ValueTypes]:
-        result = {key: False for key in command.keys()}
-        for name, _args in command.items():
-            if name == "start":
-                self.start()
-                result[name] = True
-            if name == "stop":
-                self.stop()
-                result[name] = True
-        return result
+## Configure your software
 
+1.  Go to your [machine](#prerequisites) in the [Viam app](https://app.viam.com).
+1.  Add drivers for your hardware resources. For more information, see [Supported hardware](/operate/get-started/supported-hardware/).
```

Comment:
I think we need a step in here mentioning services and/or at least mentioning non-hardware stuff, since this step specifically says "drivers for your hardware resources." Maybe something like "Add other resources such as computer vision services or non-hardware components" or similar